configuration/concepts/lab_tests.csv

```diff
@@ -98,17 +105,17 @@ e901a01d-8297-43f4-b39d-05e5ce09b708,,,Blackish,Noirâtre,Blackish,Noirâtre,,,M
-09db95e4-185c-4a22-9464-10c77dff27be,,CIEL:159994,Urine,Urine,Urine,Urine,,,Sample,N/A,,,Gravindex;Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Urine Albumin;Ascorbic Acid;Bilirubin;Urobilinogen;Urine Protein;Leucocytes;White Blood Cells;Red Blood Cells;Bacteria;Epithelial Cells;Budding Yeasts;Simple Yeasts;Mycelium;Macrophages;Kidney Cells;Leukocyte Cylinders;Granular Cylinders;Hyaline Cylinders;Erythrocyte Cylinders;Crystals;Trichomonas Vaginalis;Urinalysis,,,,,,,,
-faabf960-2025-42e1-8eed-f7825cfa6e2d,,CIEL:162614,Vaginal Swab,Vaginal Swab,Vaginal Swab,Vaginal Swab,,,Sample,N/A,,,Gram-positive Cocci;Gram-negative Cocci;Gram-positive Diplococci;Gram-negative Diplococci;Gram-negative Intracellular Diplococci;Gram-positive Bacilli;Gram-negative Bacilli;Coccobacilli;Vaginal Swab (Gram Stain);White Blood Cells;Red Blood Cells;Bacteria;Epithelial Cells;Budding Yeasts;Simple Yeasts;Mycelium;Trichomonas vaginalis;Clue cell;Vaginal Swab (Wet Mount),,,,,,,,
-7d14260b-2000-4863-9d8b-71dcb3a32f36,,CIEL:161451,Stool,Stool,Stool,Stool,,,Sample,N/A,,,Stool Consistence;Stool Appearance;Stool Colour;Stool Parasites;Stool Analysis,,,,,,,,
+2cbfd68c-b45a-4c25-b9a3-650169d87940,,CIEL:1000,Blood,Sang,Blood,Sang,,,Sample,N/A,,,508c9da4-56f9-4129-9a0e-766fb8e74f3e;Blood Glucose;Blood Group;Sickling Test;Erythrocyte Sedimentation Rate;Widal;H. Pylori;Hematocrit;White Blood Cell Count;Red Blood Cell Count;Hemoglobin;Neutrophils;Eosinophils;Basophils;Lymphocytes;Monocytes;Reticulocytes;Mean Corpuscular Volume;Mean Corpuscular Hemoglobin Concentration;Mean Corpuscular Hemoglobin;Platelets;Complete Blood Count,,,,,,,,
+09db95e4-185c-4a22-9464-10c77dff27be,,CIEL:159994,Urine,Urine,Urine,Urine,,,Sample,N/A,,,Gravindex;Urine Colour;Urine Appearance;Density;pH;Blood in urine;Nitrites;Glucose;Acetone Ketone;Urine Albumin;Ascorbic Acid;Bilirubin;Urobilinogen;Urine Protein;Urine White Blood Cells;Urine Red Blood Cells;Urine Bacteria;Urine Epithelial Cells;Urine Budding Yeasts;Urine Simple Yeasts;Urine Mycelium;Macrophages;Kidney Cells;Leukocyte Cylinders;Granular Cylinders;Hyaline Cylinders;Erythrocyte Cylinders;Crystals;Urine Trichomonas Vaginalis;Urinalysis,,,,,,,,
+faabf960-2025-42e1-8eed-f7825cfa6e2d,,CIEL:162614,Vaginal Swab,Écouvillon vaginal,Vaginal Swab,Écouvillon vaginal,,,Sample,N/A,,,Gram-positive Cocci;Gram-negative Cocci;Gram-positive Diplococci;Gram-negative Diplococci;Gram-negative Intracellular Diplococci;Gram-positive Bacilli;Gram-negative Bacilli;Coccobacilli;Vaginal Swab (Gram Stain);Swab White Blood Cells;Swab Red Blood Cells;Swab Bacteria;Swab Epithelial Cells;Swab Budding Yeasts;Swab Simple Yeasts;Swab Mycelium;Swab Trichomonas vaginalis;Clue cell;Vaginal Swab (Wet Mount),,,,,,,,

```

Comment:
"Prélèvement vaginal". That should replace "Ecouvillon" everywhere.

Reply:
Done